docs/intro.md

```diff
@@ -34,6 +34,14 @@ The admin interface is a single-page app with the entry point stored in a static
 </html>
 ```
 
+The example above uses the latest stable released version of Netlify CMS. If you want to a different release, you can explicitly declare it by editing `@~0.4`. If you want to use the latest version of Netlify CMS you can just omit the version declaration as such:
+
+```
+https://unpkg.com/netlify-cms/dist/cms.js
+https://unpkg.com/netlify-cms/dist/cms.css
+```
+Just keep in mind that the latest version might not be stable yet. Find out more about each version at the [releases page](https://github.com/netlify/netlify-cms/releases)
```

Comment:
@guayom This looks great. One thing I just found is that you can avoid pre-release (alpha, beta) versions by doing `netlify-cms@*` (with an asterisk): https://docs.npmjs.com/misc/semver#x-ranges-12x-1x-12-

Reply:
@guayom Sorry for the confusion on this, I think @erquhart mentioned that we don't want to encourage people to just use the latest version. Maybe instead of this you could just link to the list of releases and give examples how to target them. Something like this might work:
- `@~0.4` for 0.4 stable
- `@>0.5.0-beta.1` for latest 0.5 beta